src/Components/CriticalCareRecording/Others/CriticalCare__OthersEditor.res

```diff
@@ -91,6 +100,32 @@ let make = (~others, ~updateCB, ~id, ~consultationId) => {
   <div>
     <CriticalCare__PageTitle title="Others" />
     <div>
+      <div className="px-5 my-10">
+        <div className=" text-xl font-bold my-2"> {str("Bilateral Air Entry")} </div>
+        <div className="flex md:flex-row flex-col md:space-y-0 space-y-2 space-x-0 md:space-x-4">
```

Comment:
Is this field mandatory? I see excel sheet have `Y` showing its mandatory i think 😅